src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageDetailControlModel.cs

```diff
@@ -339,6 +339,14 @@ public bool IsProjectPackageReference
             }
         }
 
+        public bool IsEditableComboboxEnabled
+        {
+            get
+            {
+                return IsProjectPackageReference && !IsCentralPackageManagementEnabled;
```

Comment:
If I understand correctly, we enable editable combobox if CPM enabled.
Here it looks like opposite is happening. Please correct me if I'm wrong.